daphne_worker/src/durable/aggregate_store.rs

```diff
@@ -26,36 +24,37 @@ const OK: &str = "Ok";
 /// timestamp (in seconds) truncated by the minimum batch duration.
 #[durable_object]
 pub struct AggregateStore {
-    // TODO Write this to persistent storage instead of keeping it in memory.
-    agg_share: DapAggregateShare,
     #[allow(dead_code)]
     state: State,
 }
 
 #[durable_object]
 impl DurableObject for AggregateStore {
     fn new(state: State, _env: Env) -> Self {
-        Self {
-            agg_share: DapAggregateShare::default(),
-            state,
-        }
+        Self { state }
     }
 
     async fn fetch(&mut self, mut req: Request) -> Result<Response> {
         match (req.path().as_ref(), req.method()) {
             (DURABLE_AGGREGATE_STORE_DELETE_ALL, Method::Post) => {
-                self.agg_share.reset();
-                Response::ok(OK)
+                self.state.storage().delete_all().await?;
+                Response::empty()
```

Comment:
here and below, does `::empty` has the same meaning as `::Ok`?

Reply:
In both cases the status code is 200. This just changes the payload, which the caller ignores anyway :)